scripts/build.sh

```diff
@@ -15,6 +15,11 @@ python3 -m venv ${REPO_BASE_DIR}/target/venv
 # Install dependencies
 ${REPO_BASE_DIR}/target/venv/bin/pip install -r ${SCRIPTS_DIR}/requirements/requirements.txt
 
+# Check if the dependencies were installed successfully
+if ! which jsonschema > /dev/null 2>&1; then
```

Comment:
Use `! command -v jsonschema` instead -- see [this SO answer](https://stackoverflow.com/a/677212/7002313).